# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/gpu-alloc/.cargo-checksum.json
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/gpu-alloc/.cargo-checksum.json b/third_party/rust/gpu-alloc/.cargo-checksum.json
--- a/third_party/rust/gpu-alloc/.cargo-checksum.json
+++ b/third_party/rust/gpu-alloc/.cargo-checksum.json
@@ -1,1 +1,1 @@
-{"files":{"Cargo.toml":"2aa88a4a23b39bfaeb5b371445b04f670382efa8776db70f1a2c4f2ac3a89f04","src/allocator.rs":"c10a64551d05f3b1c108f5c3da33051450eab3311029aebce630896269e9156b","src/block.rs":"5c97435244a9785aafbacac5cc489e02430821b2dd1cda84230e3643ffb42608","src/buddy.rs":"22e3f6910b7c74cc3c3cd1122a3ca971fdd335a150b2f6fda0665717111ddac2","src/config.rs":"c392df5f029afcd2aa01b7547eed23f370bb47ee176e4b086638f1eee9cb0e14","src/error.rs":"11337a66fc32f3423a1355ffd597d2c7ea28c2b7ce07d707fd5905c330edba08","src/freelist.rs":"fd77d8f518017f4d0ef5f78256b5303aac1328bb5652d23f63357dd7d1aaec68","src/heap.rs":"03f7c623a0389263073ab48797a191ca06c5611006df7b79c9dc8615c130f495","src/lib.rs":"6fc621feade42af436404c3ed7139b76f6b6ea979309a8fc17b1593783635e53","src/linear.rs":"a7ed4ac4ec9c2025fd1e17ac35c0ba6fbf93f31c4d586692132017da00d68c94","src/slab.rs":"2df136b7e0a85eec8a198c7f807678d3a2043c46233e7ec63cd2de72fe562477","src/usage.rs":"0e4788c437718807cdd0b88301228cc27a6b8398313a91e1b70c5f2ab04dcbfb","src/util.rs":"31a496ee3407eb6b0dea255e30e154d805a0d08d777be9c06bdf56d1135bf950"},"package":"cbc1b6ca374e81862526786d9cb42357ce03706ed1b8761730caafd02ab91f3a"}
\ No newline at end of file
+{"files":{"Cargo.toml":"0ed8bb89e7cc58e25d0244a1386e38eb1c673b7659f8436f49dd4d6b2b9a49f2","src/allocator.rs":"07c7b5a9ef6242fe24388c6a05e8e9010299c91a78f88247dab27fa739e8e6a6","src/block.rs":"73cc45de56e7318fa5022fe5fe7cf7eefaa1f7d0d7a14b10c4615af5d5d06e63","src/buddy.rs":"b9e5dc10828897a7b9d6a491640c21d4cc0091c9d7fac83c76147b0c21ced71c","src/config.rs":"1851264db7576f92f3b455e4667339e1c48b7f1b88f8fae7be95c6a9badde455","src/error.rs":"50e30bccc4ba3b23d99298a65155eec45f7d91b66773e828460907ebdcb8ee41","src/freelist.rs":"92c9241d899f3e92a70b0eb5af0b557a3a76aa14c4a6d23876c3ed7857b5f15b","src/heap.rs":"347c25a8560d39d1abb807fb5c34a6a51d4786c0be24a92293f9c8217bace340","src/lib.rs":"ddbd90ad3a6e480fda6fc1bb9cc35fbd63e77152e6d2541d2bb8b59a8a156df7","src/slab.rs":"6a2b818087850bd5615b085d912490dbd46bbca0df28daa86932505b3983c64a","src/usage.rs":"fb4af7937f406cbe435bff98d66fbdb6b74131139151bde20e4183f78a5a6889","src/util.rs":"ce3fd7a278eb4d4bd030d4db5495313f56dc91b0765c394f88d126f11c2b4e75"},"package":"c481459c44304a1dfed23bd650bb3912e12c9f77d7871f86d7ed7c9730a52e79"}
\ No newline at end of file